pkg/report/templates/html/merged_report.html

```diff
@@ -59,7 +59,9 @@ <h1 class="text-3xl font-bold pb-5 pt-2 flex justify-center">Compliance Run ({{
             {{ range .Providers }}<div>
                 <label class="font-bold text-2xl">Provider {{ .Name }}</label>
                 <ul class="list-disc  list-inside">
-                    {{ range $id, $metadataText := MergedMetadataTexts . }}<li><span class="font-bold">{{ $id }}</span> {{ $metadataText }}</li>
+                    {{ $meta := MergedMetadataTexts . }}
+                    {{ $keys := SortedMapKeys $meta }}
+                    {{ range $id := $keys }}<li><span class="font-bold">{{ $id }}</span> {{ index $meta $id }}</li>
                     {{ end }}
```

Comment:
```suggestion
                    {{- $meta := MergedMetadataTexts . }}
                    {{- $keys := SortedMapKeys $meta }}
                    {{- range $id := $keys }}
                    <li><span class="font-bold">{{ $id }}</span> {{ index $meta $id }}</li>
                    {{- end }}
```
Use dash to remove unnecessary new lines from report.